dashboard/dashboard-cluster-info.md

```diff
@@ -12,11 +12,9 @@ On the cluster information page, you can view the running status of TiDB, TiKV,
 
 You can use one of the following two methods to access the cluster information page:
 
-- After logging into TiDB Dashboard, click **Cluster Info** on the left navigation menu:
+* After logging into TiDB Dashboard, click **Cluster Info** in the left navigation menu.
```

Comment:
```suggestion
* After logging in to TiDB Dashboard, click **Cluster Info** in the left navigation menu.
```

Reply:
Okay, change all `into` to `in to` in all dashboard docs.